nucliadb/src/nucliadb/search/search/chat/prompt.py

```diff
@@ -680,6 +680,7 @@ async def conversation_prompt_context(
             if field_type == "c" and paragraph.score_type in (
                 SCORE_TYPE.VECTOR,
                 SCORE_TYPE.BOTH,
+                SCORE_TYPE.BM25,
```

Comment:
If we're comparing against all score types, should we remove this check entirely?

Reply:
I think we are fine being explicit, otherwise adding a new score type will, for sure, miss an implicit condition here